vulnerabilities/scraper/npm.py

```diff
@@ -0,0 +1,105 @@
+# Author: Navonil Das (@NavonilDas)
+import semantic_version
+import re
+import json
+from urllib.request import urlopen
```

Comment:
Do you mind to sort imports?
Also can you add a standard license header as in the other files?